.github/workflows/release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         Enter-VsDevShell -VsInstallPath $vsPath -SkipAutomaticLocation -DevCmdArguments "-arch=x64 -host_arch=x64 -winsdk=10.0.19041.0"
         $Env:CC = "clang-cl"
         $Env:CXX = "clang-cl"
-        git clone --branch llvmorg-13.0.1 --depth 1 https://github.com/llvm/llvm-project.git
+        git clone --branch ${{github.ref_name}} --depth 1 https://github.com/llvm/llvm-project.git
```

Comment:
`ref_name` was your branch or tag name your pushed. for example, I push a tag named `llvmorg-13.0.1` that triggered this action run, the value of `ref_name` will be `llvmorg-13.0.1`.
following screenshot of github action's document discribed this.
![image](https://user-images.githubusercontent.com/16207250/162376070-1d30cb03-2d97-45b3-9cbf-057cf289c247.png)

Reply:
I used `ref_name` , because of we release this toolchain with tag name as same as llvmorg's according to recent releases. 
If we want to create a release manually on github website, I can add a variable that triggered by `workflow_dispatch` event to create release by github website or we can push recent change to tag we want to modify.

Reply:
Correct. We are going to use the same tag name just like llvmorg.